Add Czech sentiment dataset (#1274)
EuroEval/EuroEval: docs/datasets/czech.md

    "label": "negative"
}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 12
- Prefix prompt:

  ```
  Následují dokumenty a jejich sentiment, který může být 'pozitivní', 'neutrální' nebo 'negativní'.
  ```

- Base prompt template:

  ```
  Dokument: {text}
  Sentiment: {label}
  ```

- Instruction-tuned prompt template:

  ```
  Dokument: {text}

  Klasifikujte sentiment v dokumentu. Odpovězte pouze s 'pozitivní', 'neutrální', nebo 'negativní', a nic jiného.
  ```

- Label mapping:
  - `positive` ➡️ `pozitivní`
  - `neutral` ➡️ `neutrální`
  - `negative` ➡️ `negativní`

You can evaluate this dataset directly as follows:

```bash
euroeval --model <model-id> --dataset csfd-sentiment-mini
```

## Named Entity Recognition

### PONER

> This dataset was created [in this master thesis](https://dspace.vut.cz/items/6092e1b0-3d75-4451-8582-28573ac30404).

Reviewer: ```suggestion
This dataset was created [in this master thesis](https://hdl.handle.net/11012/213801).
```

}
```

```json
{
  "tokens": ["Konala", "se", "valná", "hromada", "Čtenářského", "spolku"],
  "labels": ["O", "O", "O", "O", "B-ORG", "I-ORG"],
}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 8
- Prefix prompt:

  ```
  Následující jsou věty a JSON slovníky s pojmenovanými entitami, které se v dané větě vyskytují.
  ```

- Base prompt template:

  ```
  Věta: {text}
  Pojmenované entity: {label}
  ```

- Instruction-tuned prompt template:

  ```
  Věta: {text}

  Identifikujte pojmenované entity ve větě. Měli byste to vypsat jako JSON slovník s klíči 'osoba', 'místo', 'organizace' a 'různé'. Hodnoty by měly být seznamy pojmenovaných entit tohoto typu, přesně tak, jak se objevují ve větě.
  ```

- Label mapping:
  - `B-PER` ➡️ `osoba`
  - `I-PER` ➡️ `osoba`
  - `B-LOC` ➡️ `místo`
  - `I-LOC` ➡️ `místo`
  - `B-ORG` ➡️ `organizace`
  - `I-ORG` ➡️ `organizace`
  - `B-MISC` ➡️ `různé`
  - `I-MISC` ➡️ `různé`

You can evaluate this dataset directly as follows:

```bash
euroeval --model <model-id> --dataset poner-mini
```

## Linguistic Acceptability

> ### ScaLA-cs

Reviewer: This should be unofficial and moved below the official one

  "label": "correct"
}
```

```json
{
  "text": "normálním průběhu sdělení se to, co je v předchozí větě jádrem, stává v další větě základem.",
  "label": "incorrect"
}
```

```json
{
  "text": "Zásady ukládají věnovat maximální pozornost hospodaření vodou a negativnímu ovlivňování životního prostředí, především čistoty vod ovzduší.",
  "label": "incorrect"
}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 12
- Prefix prompt:

  ```
  Následující jsou věty a zda jsou gramaticky správné.
  ```

- Base prompt template:

  ```
  Věta: {text}
  Gramaticky správná: {label}
  ```

- Instruction-tuned prompt template:

  ```
  Věta: {text}

  Určete, zda je věta gramaticky správná nebo ne. Odpovězte 'ano', pokud je věta správná, a 'ne', pokud není. Odpovězte pouze tímto slovem, a ničím jiným.
  ```

- Label mapping:
  - `correct` ➡️ `ano`
  - `incorrect` ➡️ `ne`

You can evaluate this dataset directly as follows:

```bash
euroeval --model <model-id> --dataset scala-cs
```

### CS-GEC

This dataset is extracted by postprocessing data from

> [this paper](https://aclanthology.org/D19-5545/). Specifically,

Reviewer: ```suggestion
[this paper](https://doi.org/10.18653/v1/D19-5545). Specifically,
```

  "question": "Jak se jmenuje produkt firmy Novell, který způsobil rozmach LAN sítí?",
  "answers": {
    "answer_start": array([969], dtype=int32),
    "text": array(["NetWare"], dtype=object)
  }
}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 4
- Prefix prompt:

  ```
  Následující texty obsahují otázky a odpovědi.
  ```

- Base prompt template:

  ```
  Text: {text}
  Otázka: {question}
  Odpověď maximálně 3 slovy: {label}
  ```

- Instruction-tuned prompt template:

  ```
  Text: {text}

  Odpovězte na následující otázku k výše uvedenému textu maximálně 3 slovy.

  Otázka: {question}
  ```

You can evaluate this dataset directly as follows:

```bash
euroeval --model <model-id> --dataset sqad-mini
```

## Knowledge

### Umimeto-qa

> This dataset offers selected questions from the <https://www.umimeto.org/> project and

Reviewer: ```suggestion
This dataset offers selected questions from the learning platform [Umimeto](https://www.umimeto.org) and
```

> has been curated by <https://nlp.fi.muni.cz/>.

Reviewer: ```suggestion
has been curated by [the NLP Centre at Masaryk University](https://nlp.fi.muni.cz).
```

  "label": "b"
}
```

```json
{
  "text": "RPSN\nVýběr:\na. roční procentní sazba nákladů\nb. roční průměrná splátka nedoplatků",
  "label": "a"
}
```

```json
{
  "text": "Jak se jmenoval slavný ruský vojevůdce v napoleonských válkách?\nVýběr:\na. Kutuzov\nb. Hannibal",
  "label": "a"
}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 5
- Prefix prompt:

  ```
  Následující jsou otázky s výběrem z více možností (s odpověďmi).
  ```

- Base prompt template:

  ```
  Otázka: {text}
  Výběr:
  a. {option_a}
  b. {option_b}
  c. {option_c}
  d. {option_d}
  Odpověď: {label}
  ```

- Instruction-tuned prompt template:

  ```
  Otázka: {text}
  Výběr:
  a. {option_a}
  b. {option_b}
  c. {option_c}
  d. {option_d}

  Odpovězte na výše uvedenou otázku pomocí 'a', 'b', 'c' nebo 'd', a nic jiného.
  ```

You can evaluate this dataset directly as follows:

```bash
euroeval --model <model-id> --dataset umimeto-qa
```

## Common-sense Reasoning

### HellaSwag-cs

This dataset is a machine translated version of the English [HellaSwag

> dataset](https://aclanthology.org/P19-1472/). The dataset was translated using

Reviewer: ```suggestion
dataset](https://doi.org/10.18653/v1/P19-1472). The dataset was translated using
```

  "label": "b"
}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 5
- Prefix prompt:

  ```
  Následující jsou otázky s výběrem z více možností (s odpověďmi).
  ```

- Base prompt template:

  ```
  Otázka: {text}
  Možnosti:
  a. {option_a}
  b. {option_b}
  Odpověď: {label}
  ```

- Instruction-tuned prompt template:

  ```
  Otázka: {text}
  Možnosti:
  a. {option_a}

>   b. {option_b}

Reviewer: Hellaswag has 4 options?

>   Odpovězte na výše uvedenou otázku pomocí 'a' nebo 'b', a nic jiného.

Reviewer: Hellaswag has 4 options?

  "target_text": "Archeologové pracují přes zimu v Národní knihovně jako o život. V poslední době zkoumali klementinské suterény pod západním křídlem bývalé jezuitské koleje a sklepní trakt mezi Studentským a Révovým nádvořím. Nejvýznamnějším nálezem je objev zbytků zdiv, které poprvé hmotně dokládají existenci zdejšího dominikánského kláštera ze 13. století"
}
```

When evaluating generative models, we use the following setup (see the
[methodology](/methodology) for more information on how these are used):

- Number of few-shot examples: 1
- Prefix prompt:

>   ```

Reviewer: ```suggestion
  ```text
```

  Následující jsou dokumenty s přiloženými souhrny.
  ```

- Base prompt template:

  ```
  Dokument: {text}
  Souhrn: {target_text}
  ```

- Instruction-tuned prompt template:

>   ```

Reviewer: ```suggestion
  ```text
```
